espressif/vscode-esp-idf-extension
# Docs/update cmds

## Description

This PR address a few different issues:

- Rename NVS Partition Editor and Partition Editor commands names.
- Add button to enable custom partition if disabled when running Partition Editor command. This will update sdkconfig in the back.
- Add `venv` validation in Setup Wizard.
- Pass current workspace folder to setup wizard save settings.

Fixes #1061
Fixes #1086

## Type of change

Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## Steps to test this pull request

Provide a list of steps to test changes in this PR and required output

1. Click on "NVS Partition Editor" command. Check that names are changed.
2. Click on "Partition Table Editor" command. Check that names are changed. If custom partition table is not enabled for current project, a notification with a button to enable should appear and if clicked the custom partition table should be enabled in sdkconfig and the partition table editor should appear.
3. Run `ESP-IDF: Configure ESP-IDF` extension. Using workspace folder as save settings location (with idf.saveScope).
4. Use setup wizard in OS without python venv installed. See that the error notification is shown.
5. Observe results.

## How has this been tested?

Manual testing based on steps above.

**Test Configuration**:
* ESP-IDF Version: 5.0.4
* OS (Windows,Linux and macOS): macOS

## Checklist
